src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,48 @@
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const allCenturies = ['16', '17', '18', '19', '20'];
+
+  function setSearchWith(params: any) {
```

Comment:
```
export type SearchParams = {
  [key: string]: string | string[] | null,
};
```

This type doesn't fit?

Reply:
check it with this approach as @kate-prakofyeva suggested